### PR TITLE
Rework of writing XFB output

### PR DESCRIPTION
**Pull request by amdrexu**
This is a follow-up change of
https://github.com/GPUOpen-Drivers/llpc/pull/2299. We don't need to pass component as an argument. Rather, this could be passed via InOutInfo. Add corresponding setter and getter.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit bd9829f0ba1dd9a541cd95d8d59fab8532e5134b
<details><summary>CTS tests (Failed: 0/208297)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 36986/69439 (53.3%)</li>
<li>Failed: 0/69439 (0.0%)</li>
<li>Not Supported: 32453/69439 (46.7%)</li><li>Warnings: 0/69439 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 37206/69417 (53.6%)</li>
<li>Failed: 0/69417 (0.0%)</li>
<li>Not Supported: 32211/69417 (46.4%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34225/69441 (49.3%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 35216/69441 (50.7%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit b55b5b739c333c9ca1747667b36a976f10eb14c6
<details><summary>CTS tests (Failed: 0/208295)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 36988/69437 (53.3%)</li>
<li>Failed: 0/69437 (0.0%)</li>
<li>Not Supported: 32449/69437 (46.7%)</li><li>Warnings: 0/69437 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 37206/69417 (53.6%)</li>
<li>Failed: 0/69417 (0.0%)</li>
<li>Not Supported: 32211/69417 (46.4%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34225/69441 (49.3%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 35216/69441 (50.7%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 40ecd6950f19e7638ea23d27d3cba5c538ad5f2b
<details><summary>CTS tests (Failed: 206/208295)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 36988/69437 (53.3%)</li>
<li>Failed: 0/69437 (0.0%)</li>
<li>Not Supported: 32449/69437 (46.7%)</li><li>Warnings: 0/69437 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 37103/69417 (53.4%)</li>
<li><details><summary>Failed: 103/69417 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.dmat3.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.dvec2.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.vec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.dvec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.ivec2.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.ivec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.mat2x4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.vec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.instance_array_basic_type.mat3x4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.multi_nested_struct.std_geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.41
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.63
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.80
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.92
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.97
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.10
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.35
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.40
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.51
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.52
Stack trace: Fail
...
```
</p></details></li>

<li>Not Supported: 32211/69417 (46.4%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34122/69441 (49.1%)</li>
<li><details><summary>Failed: 103/69441 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.dmat3.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.dvec2.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.2_level_array.vec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.dvec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.ivec2.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.ivec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.mat2x4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.3_level_array.vec4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.instance_array_basic_type.mat3x4.geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.multi_nested_struct.std_geometry
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.41
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.63
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.80
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.92
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_instance_array.97
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.10
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.35
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.40
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.51
Stack trace: Fail

FAILURE: dEQP-VK.transform_feedback.fuzz.random_geometry.all_missing.52
Stack trace: Fail
...
```
</p></details></li>

<li>Not Supported: 35216/69441 (50.7%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/patch/PatchInOutImportExport.cpp [hunk 1]
```diff
@@ -1024,11 +1027,6 @@ void PatchInOutImportExport::visitCallInst(CallInst &callInst) {
           patchMeshGenericOutputExport(output, loc, locOffset, elemIdx, vertexOrPrimitiveIdx, isPerPrimitive, builder);
           break;
         }
-        case ShaderStageFragment: {
-          assert(callInst.arg_size() == 3);
-          llvm_unreachable("Fragment shader export should have been handled by the LowerFragColorExport pass");
-          break;
-        }
```
Comment (nhaehnle) [inline lgc/patch/PatchInOutImportExport.cpp:1031]: Seems like this is an unrelated change?

Comment (amdrexu) [reply lgc/patch/PatchInOutImportExport.cpp:1031]: Oh. I mistakenly changed this code. Should be in another irrelevant PR.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 645f9167ff7a905a500b43e0fb3afdc691162f7c
<details><summary>CTS tests (Failed: 67/208297)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 37180/69439 (53.5%)</li>
<li>Failed: 0/69439 (0.0%)</li>
<li>Not Supported: 32259/69439 (46.5%)</li><li>Warnings: 0/69439 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 6571/69417 (9.5%)</li>
<li><details><summary>Failed: 67/69417 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.api.buffer_view.create.dedicated_alloc.uniform.a8b8g8r8_sint_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.buffer_view.create.suballocation.uniform.r8g8b8_snorm
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.2d.eac_r11g11_snorm_block.r4g4_unorm_pack8.linear_general_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.2d.r8g8b8_snorm.r4g4_unorm_pack8.linear_optimal_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.image_to_image.dimensions.src4x32768_dst4x32768.r4g4_unorm_pack8.r8_srgb.general_optimal
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.resolve_image.diff_image_size.dst_256_256_11_4_bit
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.blit_image.all_formats.generate_mipmaps.from_previous_level.layercount_1.a2b10g10r10_sscaled_pack32.general_general_linear
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.multi.a8b8g8r8_uint_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.multi.r8_uint
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.random.r8g8b8a8_srgb
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.single.d32_sfloat
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.multiple_layers.r5g6b5_unorm_pack16_33x128
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.a2r10g10b10_unorm_pack32_200x180
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.b8g8r8a8_srgb_1x33
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.r8_sint_33x128
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.d16_unorm_s8_uint
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.g10x6_b10x6_r10x6_3plane_444_unorm_3pack16
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.r12x4g12x4b12x4a12x4_unorm_4pack16
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.r32_uint
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.r8g8_uscaled
Stack trace: Flake
...
```
</p></details></li>

<li>Not Supported: 62779/69417 (90.4%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34417/69441 (49.6%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 35024/69441 (50.4%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8265f1f190eeff5ec77536c5b290d407998c0eeb
<details><summary>CTS tests (Failed: 189/208297)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 37180/69439 (53.5%)</li>
<li>Failed: 0/69439 (0.0%)</li>
<li>Not Supported: 32259/69439 (46.5%)</li><li>Warnings: 0/69439 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 7120/69417 (10.3%)</li>
<li><details><summary>Failed: 189/69417 (0.3%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.api.buffer.dedicated_alloc.transfer_src.transfer_dst.uniform.index.create.zero
Stack trace: Flake

FAILURE: dEQP-VK.api.buffer_marker.graphics.default_mem.top_of_pipe.sequential.64
Stack trace: Flake

FAILURE: dEQP-VK.api.command_buffers.allocate_single_primary
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.2d.r4g4_unorm_pack8.r4g4_unorm_pack8.optimal_optimal_linear
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.3d.r8g8_sscaled.r8g8_sscaled.general_general_linear_stripes_y
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.generate_mipmaps.from_previous_level.layercount_6.a8b8g8r8_sint_pack32.optimal_optimal_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.core.blit_image.all_formats.color.2d.a4b4g4r4_unorm_pack16.b5g5r5a1_unorm_pack16.optimal_general_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.core.blit_image.all_formats.color.2d.r32g32_sfloat.r8g8b8a8_sscaled.optimal_optimal_cubic
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.core.image_to_image.all_formats.color.2d.bc3_srgb_block.bc3_unorm_block.optimal_optimal
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.core.image_to_image.dimensions.src32768x4_dst32768x4.b8g8r8_srgb.b8g8r8_srgb.optimal_optimal
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.core.image_to_image.dimensions.src4096x4_dst4096x4.r4g4b4a4_unorm_pack16.r4g4b4a4_unorm_pack16.optimal_general
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.blit_image.all_formats.color.3d.etc2_r8g8b8a1_srgb_block.etc2_r8g8b8a1_srgb_block.general_general_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.blit_image.all_formats.color.3d.r4g4b4a4_unorm_pack16.r4g4b4a4_unorm_pack16.linear_general_linear_stripes_y
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.image_to_image.all_formats.color.3d.r4g4_unorm_pack8.r4g4_unorm_pack8.optimal_general
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.image_to_image.dimensions.src4096x4_dst4096x4.r8g8b8a8_unorm.r32_sfloat.optimal_general
Stack trace: Flake

FAILURE: dEQP-VK.api.external.fence.opaque_win32.signal_import_temporary
Stack trace: Flake

FAILURE: dEQP-VK.api.external.semaphore.opaque_win32_kmt.export_import_signal_wait_permanent
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.in_render_pass.r8g8_uscaled
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.multi.a2r10g10b10_unorm_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.core.partial_clear_color_attachment.multiple_layers.r5g5b5a1_unorm_pack16_64x11
Stack trace: Flake
...
```
</p></details></li>

<li>Not Supported: 62108/69417 (89.5%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34417/69441 (49.6%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 35024/69441 (50.4%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8265f1f190eeff5ec77536c5b290d407998c0eeb
<details><summary>CTS tests (Failed: 63/208297)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 37116/69439 (53.5%)</li>
<li>Failed: 0/69439 (0.0%)</li>
<li>Not Supported: 32323/69439 (46.5%)</li><li>Warnings: 0/69439 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 11120/69417 (16.0%)</li>
<li><details><summary>Failed: 63/69417 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.api.buffer_view.create.dedicated_alloc.uniform.a8b8g8r8_sint_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.buffer_view.create.suballocation.uniform.r8g8b8_snorm
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.2d.eac_r11g11_snorm_block.r4g4_unorm_pack8.linear_general_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.blit_image.all_formats.color.2d.r8g8b8_snorm.r4g4_unorm_pack8.linear_optimal_nearest
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.image_to_image.dimensions.src4x32768_dst4x32768.r4g4_unorm_pack8.r8_srgb.general_optimal
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.copy_commands2.resolve_image.diff_image_size.dst_256_256_11_4_bit
Stack trace: Flake

FAILURE: dEQP-VK.api.copy_and_blit.dedicated_allocation.blit_image.all_formats.generate_mipmaps.from_previous_level.layercount_1.a2b10g10r10_sscaled_pack32.general_general_linear
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.multi.a8b8g8r8_uint_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.multi.r8_uint
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.random.r8g8b8a8_srgb
Stack trace: Flake

FAILURE: dEQP-VK.api.granularity.single.d32_sfloat
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.multiple_layers.r5g6b5_unorm_pack16_33x128
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.a2r10g10b10_unorm_pack32_200x180
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.b8g8r8a8_srgb_1x33
Stack trace: Flake

FAILURE: dEQP-VK.api.image_clearing.dedicated_allocation.clear_color_attachment.single_layer.r8_sint_33x128
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.d16_unorm_s8_uint
Stack trace: Flake

FAILURE: dEQP-VK.api.info.format_properties.g10x6_b10x6_r10x6_3plane_444_unorm_3pack16
Stack trace: Flake

FAILURE: dEQP-VK.api.info.image_format_properties.1d.optimal.a2r10g10b10_sscaled_pack32
Stack trace: Flake

FAILURE: dEQP-VK.api.info.image_format_properties.3d.linear.astc_12x12_srgb_block
Stack trace: Flake

FAILURE: dEQP-VK.api.info.image_format_properties2.3d.optimal.r8_srgb
Stack trace: Flake
...
```
</p></details></li>

<li>Not Supported: 58234/69417 (83.9%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 34417/69441 (49.6%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 35024/69441 (50.4%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8265f1f190eeff5ec77536c5b290d407998c0eeb
<details><summary>CTS tests (Failed: 0/208295)</summary><ul><li>Built with version 1.3.3.1</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 40209/69437 (57.9%)</li>
<li>Failed: 0/69437 (0.0%)</li>
<li>Not Supported: 29228/69437 (42.1%)</li><li>Warnings: 0/69437 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 40225/69417 (57.9%)</li>
<li>Failed: 0/69417 (0.0%)</li>
<li>Not Supported: 29192/69417 (42.1%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 36765/69441 (52.9%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 32676/69441 (47.1%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8265f1f190eeff5ec77536c5b290d407998c0eeb
<details><summary>CTS tests (Failed: 0/208297)</summary><ul><li>Built with version 1.3.3.1</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 40199/69439 (57.9%)</li>
<li>Failed: 0/69439 (0.0%)</li>
<li>Not Supported: 29240/69439 (42.1%)</li><li>Warnings: 0/69439 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 40225/69417 (57.9%)</li>
<li>Failed: 0/69417 (0.0%)</li>
<li>Not Supported: 29192/69417 (42.1%)</li><li>Warnings: 0/69417 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx10</summary><ul>

<li>Passed: 36765/69441 (52.9%)</li>
<li>Failed: 0/69441 (0.0%)</li>
<li>Not Supported: 32676/69441 (47.1%)</li><li>Warnings: 0/69441 (0.0%)</li>
</ul></details>


</ul></details>

**Review by nhaehnle — APPROVED** [review]
